### ISSUE TITLE
The tab feature

### ISSUE THREAD

**Issue by SanmayJoshi** [veniosg/Dir#49]
Hey George,

Maybe, Dir could have tabs? If you think it's a good idea, I had a small suggestion. Tabs as in ES explorer make it not much usable and often create confusions. So, maybe, tabs as in, say, UC mini browser might play well. Like adding a square icon, with tab count, on screen somewhere(as you seem fit) may do the trick.

Thanks.

**Comment by veniosg** [comment]
Either tabs or just multiple panes sounds good. Given how non-trivial this change is, I'm adding this to the backlog for potentially interested contributors to show interest and pick up the discussion from there. 👍

**Comment by bdtoole** [comment]
I think in file managers like this, multiple panes look a bit better from a simplicity standpoint, and are arguably just as functional as tabs. A good example of this is Amaze, which has two panes that you can swipe left/right between, rather than adding "clutter" to the interface via visible tabs.

Just my 2c, though - others may disagree!

**Comment by veniosg** [comment]
I tend to fall somewhere in between. This being a power user feature, it doesn't have to be obvious but there should be some indication that you can swipe left/right to get to the other pane to avoid confusion on accidental switches. 
Additionally this should be kept as opt-in, given how the current navigation animation effectively suggests that swiping could navigate back (which is not the case) and can lead to further confusion.

**Comment by bdtoole** [comment]
I agree there should be an indicator - currently, Amaze uses two small dots on the lower portion of the screen.

Good point about the current navigation functionality with it, as well. In this case, tabs might be a better implementation than panes, just to preserve the existing nav functionality.

**Comment by veniosg** [comment]
Maybe a slim drag indicator on the left/right (depending on which pane you're on) would be good enough and provide similar ease of use without being confusing with the nav. (I should really check how amaze handles this)
